packages/server/test/index.ts

```diff
@@ -345,3 +355,61 @@ test('find package usages', async t => {
   await storeCtrl.close()
   t.end()
 })
+
+test('server should only allow POST', async (t) => {
+  const port = 5813
+  const hostname = '127.0.0.1'
+  const remotePrefix = `http://${hostname}:${port}`
+  const storeCtrlForServer = await createStoreController()
+  const server = createServer(storeCtrlForServer, {
+    hostname,
+    port,
+  })
+
+  t.ok(await isPortReachable(port), 'server is running')
+
+  // Try various methods (not including POST)
+  const methods = ['GET', 'PUT', 'PATCH', 'DELETE', 'OPTIONS']
+
+  for (let method of methods) {
+    console.log(`Testing HTTP ${method}`)
```

Comment:
use [t.comment()](https://www.npmjs.com/package/tape#tcommentmessage) instead of `console.log`